css/main.css

```diff
@@ -325,3 +325,125 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  margin: 120px 0 0 0;
+}
+
+.contact {
+  padding: 0 24px;
+}
+
+.description-contact {
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  margin: 0 0 73px 0;
+  color: #172b4d;
+}
+
+.contact-inputs {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] **[OPTIONAL]**: Very nice use of `flexbox` to style your layout! 💯 However I noticed that you used it in many selectors all over your CSS stylesheet. It would be great if you can find a way of reducing the repetition of code 💻. This will help you keep your code clean, readable, and following the D.R.Y principle. 

There are a lot of ways of avoiding code repetition, feel free to use the one that you like the most 💯 Here's one of them:

You could define a `flex` class:
```CSS
.flex {
   display: flex;
}
```

And then attach it wherever you need a flex container:
```HTML
<div class="flex"></div>
```